src/js/app.js

```diff
@@ -631,6 +631,8 @@
                     }))
                 ;
 
+                let username = document.querySelector('div.user').textContent;
+                username = username.substr(0, username.lastIndexOf(' '));
```

Comment:
One year later...
I wasn't aware of all these sponsorship differentiations, so I suppose the best option is to rely on the **member_id**. The only page where I was able to retrieve it was the **Settings page** where you can choose to be named with your user ID.
If you can request the page and store the ID as a setting, it will probably do the job.

My proposal was just a quick improvement and now it's becoming a real feature. I'm not sure I will be able to follow with all this.

Reply:
> My proposal was just a quick improvement and now it's becoming a real feature. I'm not sure I will be able to follow with all this.

No worries! Was partially talking to myself. I appreciate the effort either way, and was happy to further follow-up on it. Thanks again, and you should see your feature (with extras) in an upcoming edition!

Note to self: As mentioned in the PR comments, found a fix in `owner_id` .